client/src/pages/OfferHelp.js

```diff
@@ -65,6 +66,7 @@ const Step1 = (props) => {
           {Object.entries(answers).map(([answer, checked], i) => (
             <WizardCheckboxItem
               key={i}
+              id={GTM.offerHelp.prefix + GTM.offerHelp.volunteer + i}
```

Comment:
Not working as expected just yet. I will take a look soon before resolving some other issues.